typespec-extension/src/common/client.ts

```diff
@@ -50,9 +48,6 @@ export class ServiceVersion extends Metadata {
   }
 }
 
-export class ObjectScheme extends ObjectSchema {
+export interface CrossLanguageDefinition {
   crossLanguageDefinitionId?: string;
-  constructor(name: string, description: string, objectInitializer?: DeepPartial<ObjectSchema>) {
-    super(name, description, objectInitializer);
-  }
```

Comment:
This part of change is not related to multipart.

I was just surprised to see we had `ObjectScheme` and `ObjectSchema`.

I had plan to refactor the representation of `crossLanguageDefinitionId` in code model, but not having time though.

Reply:
Previously `ObjectScheme` is just used for adding `crossLanguageDefinitionId` ?

Reply:
That's my guess too.

For now, I changed to `(schema as CrossLanguageDefinition).crossLanguageDefinitionId = ...`

Future plan is to move this to `schema.language.default.crossLanguageDefinitionId` but that would involve Java changes.